packages/dx-react-chart/demos/area/step-area.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Step Area
+
+The demo shows how to customize an [Area Series](../../docs/reference/area-series.md) to display a Step Area series and demonstrates legend item customization.
```

Comment:
customize and customization
The demo shows how to modify an [Area Series](../../docs/reference/area-series.md) to display a Step Area series and demonstrates legend item customization.